NSS-Day-Cohort-42/rare-API-media-moguls
# Create in Category Viewset

#### Changes Made
1. Added 'create' method to category.py in views

#### Steps to Review
1. Checkout this branch locally and run the application.

```
git fetch --all
git checkout sm-createcategory
```

4. Initialize virtual environment, and run the server.

```
pipenv shell
python manage.py runserver
```
5. Test functionality.
> Go to Postman 
>Run a GET with `http://localhost:8000/categories` to ensure that you can correctly get all categories. 
>Run a POST with the same URL but before hitting send, go to the body tab and insert this information:
`{
    "label": "unicorns"
}`
>Double check that you have selected "raw" and JSON under the body tab
>Once you hit send, you should see the updated object with an id. 
>If you do a GET command for all categories it should be added to the list! 

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
